assets/parquet/ingest_json_string.py

```
@@ -0,0 +1,29 @@
+import sys
+
+INPUT_FILE = sys.argv[1]
+OUTPUT_FILE = "/home/hive-data/bench_schema/bench_table/mongod.parquet"
+BATCH_SIZE = 10000
+
+schema = pa.schema([("line", pa.string())])
+
+#with pq.ParquetWriter(OUTPUT_FILE, schema) as writer:
+with pq.ParquetWriter(OUTPUT_FILE, schema, compression="ZSTD", compression_level=3) as writer:
+    batch = []
+    with open(INPUT_FILE, "r") as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)